docs/remote_dcrwallet.md

```diff
@@ -81,15 +80,18 @@ node = dcrw
 # Replace 19221 for the correct port.
 dcrwallet.grpchost = 127.0.0.1:19221
 
-# This is a copy of the standard wallet rpc.cert file.
-dcrwallet.certpath = rpc.cert
+# This is a copy of the standard dcrwallet rpc.cert file.
+# if you are running dcrwallet and dcrlnd on the same host
+# this will be in ~/.dcrwallet/rpc.cert by default.
+dcrwallet.certpath = /path/to/rpc.cert
 
 # Account number from which the LN keys will be derived. DO NOT CHANGE after the
 # dcrlnd wallet is setup. The account must already exist.
 dcrwallet.accountnumber = 1
 
 # Replace for the full path to the client.key and client.cert files previously
 # created.
-dcrwallet.clientkeypath = client.key
-dcrwallet.clientcertpath = client.cert
+dcrwallet.clientkeypath = /path/to/client-ca.key
+dcrwallet.clientcertpath = /path/to/client-ca.cert
 ```
+On startup, dcrlnd will still prompt to unlock the wallet even if the dcrwallet instance has it unlocked, you have to use **dcrlncli unlock** as you would if it was the embedded wallet.
```

Comment:
Sentence is too long and I think the reference to the embedded wallet mode is unnecessary. A better wording would be:

> On startup, dcrlnd will still prompt to unlock the wallet even if the `dcrwallet` instance is unlocked. Execute `dcrlncli unlock`, typing the wallet's **private passphrase** so that `dcrlnd` extracts the keys needed for its operation.